pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = ["setuptools>=43.0.0", "setuptools_scm[toml]>=6.2", "wheel",
+            "oldest-supported-numpy"]
```

Comment:
Since this package has no C or cython code, no need to have `numpy` or `oldest-supported-numpy` as a build dependency here.

Reply:
Right!